crypto/ec/ecp_nistz256.c

```diff
@@ -1519,7 +1664,12 @@ const EC_METHOD *EC_GFp_nistz256_method(void)
         0,                                          /* field_div */
         ec_GFp_mont_field_encode,
         ec_GFp_mont_field_decode,
-        ec_GFp_mont_field_set_to_one
+        ec_GFp_mont_field_set_to_one,
+#if defined(__x86_64) || defined(__x86_64__)
```

Comment:
This glue does not look quite right - I can imagine at some point we may have this for other than x86_64, too. Perhaps we can integrate a define (`ECP_NISTZ256_ASM_MOD_INVERSE` or some such) into the Configure cascades? (@dot-asm may have a suggestion?)

Also, drive-by and I haven't really thought about it myself but wdyt about the suggestion to disentangle this from nistz and engage it for ecp_nistp256 as well? Worth it?